k8s.gcr.io/images/k8s-staging-kubernetes/images.yaml

```diff
@@ -315,6 +315,7 @@
     "sha256:9b9987c6a07813074a69c7c9aa2a603590ea3a3097be0c3b44e2d57c54ff5c0b": ["v1.26.1-rc.0"]
     "sha256:e8fca63a0a9ae8da4a666ba6d4298b430d5775386568731693f922832ea86c11": ["v1.26.2"]
     "sha256:42b7bbe05edcc15190715eef5cb533c51c03e6c71db6c0e8db85e9fd215ce482": ["v1.26.2-rc.0"]
+    "sha256:acfc9bf377a7480867b6310a60b4aeebaff69cee5bb5885ba95d9be5bd343b08": ["v1.26.3"]
```

Comment:
Can you rerun kpromo with v3.5.1? We moved the manifests folder from k8s.gcr.io folder to registry.k8s.io folder.

Reply:
https://github.com/kubernetes/k8s.io/pull/4948#issuecomment-1473146353